chart/values.schema.json

```diff
@@ -6931,6 +6931,14 @@
             "x-docsSection": "Airflow",
             "additionalProperties": false,
             "properties": {
+                "mountPath": {
+                    "description": "Where dags volume will be mounted. Works for both `persistence` and `gitSync`. If not specified, dags mount path will be set to `$AIRFLOW_HOME/dags`",
```

Comment:
Maybe we should also take this into account in the `airflow_dags` helper?

Reply:
@jedcunningham good point! I personally prefer to set dags_folder manually, but the default value can be adjusted too.